virttest/bootstrap.py

```diff
@@ -360,6 +365,20 @@ def _forced_or_detected(forced, detected):
         cfg.write("                            - @%s:\n" % release)
         cfg.write("                                variants:\n")
         cfg.write("                                    - @%s:\n" % arch)
+        cfg.write("variants:\n")
+        cfg.write("    - @Hostvendor:\n")
```

Comment:
We'd better add the word `CPU` to these variants to indicate that they are used for the identification of cpu devices.

Btw, I suggest using CapWords convention.

Reply:
makes sense for usability, will change it.